.github/workflows/release.yml

```diff
@@ -0,0 +1,39 @@
+name: Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      imageTag:
+        description: "The tag of the image that you want to release"
+        required: true
+      version:
+        description: "The version that will be released"
+        required: true
+
+env:
+  GHCR_REPO: ghcr.io/macstadium/orka-github-runner
+
+jobs:
+  release:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Get image manifest
+        id: image_manifest
+        run: |
+          manifest=$(curl -s -X GET https://ghcr.io/v2/macstadium/orka-github-runner/manifests/${{ github.event.inputs.imageTag }} -u ${{ github.actor }}:${{ secrets.GITHUB_TOKEN }})
+          echo "manifest=${manifest}" >> $GITHUB_OUTPUT
+
+      - name: Fail if the provided image tag is invalid
+        if: "contains(steps.image_manifest.outputs.manifest, 'unknown')"
+        run: exit 1
```

Comment:
Why do we need this? Wasn't this logic from the old workflow needed to copy the image from the private package to the public one?